tests/features/social-media-follow.feature

```diff
@@ -31,11 +31,14 @@ Feature: Social media follow paragraph.
     And I fill in the 3rd "URL" field with "http://twitter.com"
     And I fill in the 3rd "Link text" with "2nd Twitter"
     And I select "Twitter" from the 3rd "Link type"
+    And I fill in the 4th "URL" field with "https://europa.eu/european-union/contact/social-networks_en"
```

Comment:
I'm not completely happy with this step. As it is, it seems that we are filling another social link and indeed when we check the rendered values nothing implies it isn't another social link. Could we make sure we are filling the correct link by using the same format as in oe_content? (see https://github.com/openeuropa/oe_content/blob/EPIC-Event/tests/features/event.feature#L197)